src/util/apis/generated/operations/buildOperations.js

```diff
@@ -860,7 +860,7 @@ BuildOperations.prototype.getCommits = function (shaCollection, ownerName, appNa
   // Construct URL
   var baseUrl = this.client.baseUri;
   var requestUrl = baseUrl + (baseUrl.endsWith('/') ? '' : '/') + 'v0.1/apps/{owner_name}/{app_name}/commits/batch/{sha_collection}';
-  requestUrl = requestUrl.replace('{sha_collection}', encodeURIComponent(shaCollection));
```

Comment:
This is not a real fix, it is merely a temporary workaround. We need to remove it before pushing this PR. Ideally the rest of the changes from this commit will not require any updates, once buildOperations.getCommits will be fixed for a multiple SHAs.